tests/manage/pv_services/test_delete_project_with_pvc_and_pods.py

```diff
@@ -0,0 +1,163 @@
+"""
+The purpose of this test case is to delete at least 2 projects:
+One project with just pvcs and the other with pvcs attached to a pod
+"""
+import logging
+import time
+import pytest
+from itertools import cycle
+
+from ocs_ci.framework.testlib import ManageTest, tier1
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.ocs.cluster import CephCluster
+from ocs_ci.ocs.resources.ocs import OCS
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@pytest.mark.polarion_id("OCS-278")
+@pytest.mark.parametrize(
+    argnames=["pvcs_num"],
+    argvalues=[
+        pytest.param(
```

Comment:
Looks good. 
We'll need each params permutation to be reported with a different polarion_id.
@keesturam @nehaberry wdyt, should we have a polarion case for the scale permutation too?